### ISSUE TITLE
Some enhancement ideas

### ISSUE THREAD

**Issue by realspinelle** [Moonlight-Panel/Moonlight#281]
- [] Add a way to add/edit languages on the panel
- []  Add a language selector top right ( like most website ) ( need to be saved for the next idea )
- [] Add a way to edit mail template on the panel ( but per language )
- [] Add a way to send a notification to some / 1 client even if he's offline
- [] Be able to delete a user maybe by a permission
- [] Add a maintenance toggle per node & programmable maintenance and it shows on all servers in that node a notification that says that this node will be in maintenance in x range also maintenance turn on automatically at the time set

**Comment by Marcel-Baumgartner** [comment]
I took note of the enhancements. I will report by progress in this issue ^^

**Comment by realspinelle** [comment]
Is there some things that is not a good idea or ?

**Comment by Marcel-Baumgartner** [comment]
No, every bullet point is a good enhancement and will be implemented in the future ^^
